src/test/java/dev/ebullient/fc5/Fc5ConvertIT.java

```diff
@@ -0,0 +1,127 @@
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
+
+import dev.ebullient.test.QuarkusMainLauncher;
+import dev.ebullient.test.QuarkusMainLauncher.LaunchPaths;
+import dev.ebullient.test.QuarkusMainLauncher.LaunchResult;
+import dev.ebullient.test.QuarkusMainTest;
+import dev.ebullient.test.QuarkusMainTest.Launch;
+
+@QuarkusMainTest
+public class Fc5ConvertIT {
+    final static Path PROJECT_PATH = Paths.get(System.getProperty("user.dir")).toAbsolutePath();
+
+    @BeforeEach
+    void cleanUp(TestInfo info, LaunchPaths paths) {
+        Path targetDir = paths.getProjectBuildPath().resolve(info.getDisplayName());
+        deleteDir(targetDir);
+    }
+
+    @Test
+    @Launch(args = {})
+    void testThings() {
+        System.out.println("Life is good");
+    }
+
+    @Test
+    @Launch(args = { "--help" })
+    void testCommandHelp(LaunchResult result) {
+        System.out.println(result.getOutputStream().stream().collect(Collectors.joining("\n")));
+        Assertions.assertTrue(result.getOutputStream().stream()
+                .anyMatch(x -> x.startsWith("Usage: fc5-convert [-hvV] [<input>...] [COMMAND]")),
+                "Result should contain the CLI help message. Found: " + result);
+    }
+
+    @Test
+    @Launch(args = { "obsidian", "--help" })
+    void testObsidianCommandHelp(LaunchResult result) {
+        System.out.println(result.getOutputStream().stream().collect(Collectors.joining("\n")));
+        Assertions.assertTrue(result.getOutputStream().stream()
+                .anyMatch(x -> x.startsWith("Usage: fc5-convert obsidian [-hvV] -o=<outputPath> [<input>...]")),
+                "Result should contain the CLI help message. Found: " + result);
+    }
+
+    @Test
+    @Launch(args = { "transform", "--help" })
+    void testTransformCommandHelp(LaunchResult result) {
+        System.out.println(result.getOutputStream().stream().collect(Collectors.joining("\n")));
+        Assertions.assertTrue(result.getOutputStream().stream()
+                .anyMatch(x -> x.startsWith("Usage: fc5-convert transform [-hvV] -o=<outputPath> [-t=<xsltFile>]")),
+                "Result should contain the CLI help message. Found: " + result);
+    }
+
+    @Test
+    @Launch(args = { "validate", "--help" })
+    void testValidateCommandHelp(LaunchResult result) {
+        System.out.println(result.getOutputStream().stream().collect(Collectors.joining("\n")));
+        Assertions.assertTrue(result.getOutputStream().stream()
+                .anyMatch(x -> x.startsWith("Usage: fc5-convert validate [-hvV] [--collection | [--compendium] |")),
+                "Result should contain the CLI help message. Found: " + result);
+    }
+
+    @Test
+    void testFc5Xml(TestInfo info, QuarkusMainLauncher launcher, LaunchPaths paths) {
+        final Path targetDir = paths.getProjectBuildPath().resolve(info.getDisplayName());
+        LaunchResult result;
+
+        result = launcher.runQuarkusMain(PROJECT_PATH,
```

Comment:
any reason why its `runQuarkusMain` and not just `run` ?

Reply:
because that's what I thought was more explicit (go find the QuarkusMain and run it) .. naming things is hard